src/Raven.Client/Http/HttpCache.cs

```diff
@@ -27,9 +26,11 @@ public class HttpCache : IDisposable, ILowMemoryHandler
         /// </summary>
         public int NumberOfItems => _items.Count;
 
+        public long MaxSize { get; set; }
```

Comment:
Maybe it would be better to expose it via Conventions?

Reply:
I think we will have to change the HttpCache in requestExecutor if we want to expose it via Conventios. but I am not sure.

Reply:
We could introduce MaxHttpCacheSize (or MaxRequestExecutorCacheSize ?) in DocumentConventions and in RequestExecutor constructor initialize the HttpCache with that value. Should be relatively easy to do and the configurability level would be much higher.

Reply:
Done